chromium_src/chrome/updater/DEPS

```diff
@@ -0,0 +1,6 @@
+from brave_chromium_utils import inline_file
+inline_file("//chrome/updater/DEPS", globals(), locals())
```

Comment:
this inline shouldn't be necessary here as it's done automatically for `chromium_src/**/DEPS` files.